cpu/samd5x/Makefile.features

```diff
@@ -1,8 +1,8 @@
 CPU_CORE = cortex-m4f
 
 FEATURES_PROVIDED += periph_hwrng
-FEATURES_PROVIDED += backup_ram
 FEATURES_PROVIDED += cortexm_mpu
+FEATURES_PROVIDED += periph_backup_ram
```

Comment:
One could argue that 'backup RAM' and MPU are both CPU features, not peripherals.

Reply:
Can I then rename the test to `tests/backup_ram`, put `backup_ram.c` in `cpu/stm32/` and `backup_ram.h` in `stm32/include/` ?

Reply:
Ah you are right, it's cleaner that way.
Also existing implementations didn't require `FEATURES_REQUIRED = backup_ram`, backup ram was always powered, so this shouldn't break existing applications.